composeApp/src/androidMain/kotlin/it/fast4x/rimusic/ui/screens/player/components/controls/Essential.kt

```diff
@@ -346,6 +348,8 @@ fun ControlsEssential(
 
     var queueLoopType by rememberPreference(queueLoopTypeKey, defaultValue = QueueLoopType.Default)
     val playerBackgroundColors by rememberPreference(playerBackgroundColorsKey,PlayerBackgroundColors.BlurredCoverColor)
+    var jumpPrevious by rememberPreference(jumpPreviousKey,"3")
```

Comment:
Why use a text field if after convert in toInt()... use directly int...

Reply:
Integer type can't be blank and while entering the value in the text field if the text is empty it will throw errors.

So I'm using string and converting it to integer later so that the text field can be empty.

Pls improve it the way you want

Reply:
No ok, in the future if necessary.